salt/orchestrate/edx/backup.sls

```diff
@@ -88,6 +107,33 @@ execute_enabled_backup_scripts:
         - salt: deploy_backup_instance_to_{{ ENVIRONMENT }}
         - salt: mount_backup_drive
 
+unmount_backup_drive:
+  salt.function:
+    - tgt: 'G@roles:backups and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: state.single
+    - arg:
+        - mount.unmounted
+    - kwarg:
+        name: /backups
+        device: /dev/{{ salt.grains.get('ec2:block_device_mapping:ebs2') }}
```

Comment:
This needs a require statement to tie it to the success state of the backup scripts.